Formula/rist.rb

```diff
@@ -0,0 +1,37 @@
+class Rist < Formula
+  desc "Reliable Internet Stream Transport (RIST)"
+  homepage "https://code.videolan.org/rist/"
+  url "https://code.videolan.org/rist/librist/-/archive/v0.2.6/librist-v0.2.6.tar.gz"
+  sha256 "88b35b86af1ef3d306f33674f2d9511a27d3ff4ec76f20d3a3b3273b79a4521d"
+  license "BSD-2-Clause"
+  head "https://code.videolan.org/rist/librist.git", branch: "master"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "cjson"
+  depends_on "cmocka"
+  depends_on "mbedtls"
+
+  def install
+    mkdir "build"
+    system "meson", "build", "."
+    system "ninja", "-C", "build"
+    bin.mkpath
+    bin.install "build/tools/rist2rist"
+    bin.install "build/tools/ristreceiver"
+    bin.install "build/tools/ristsender"
+    bin.install "build/tools/ristsrppasswd"
+    lib.mkpath
+    lib.install Dir["build/librist*.dylib"]
+    inc = include/"librist"
+    inc.mkpath
+    inc.install "build/include/vcs_version.h"
+    inc.install "build/include/librist/version.h"
+    inc.install Dir["include/librist/*.h"]
+    system "chmod 444 #{include}/librist/*.h"
```

Comment:
```suggestion
    system "meson", *std_meson_args, "build"
    system "ninja", "install", "-C", "build"
```
Does this not work?

Reply:
The test on 11-arm64 failed:
~~~
==> Testing rist
==> /opt/homebrew/Cellar/rist/0.2.6/bin/ristsender 2>&1
Error: test failed
sh: line 1: 49190 Abort trap: 6           /opt/homebrew/Cellar/rist/0.2.6/bin/ristsender 2>&1
dyld: Library not loaded: @rpath/librist.4.dylib
  Referenced from: /opt/homebrew/Cellar/rist/0.2.6/bin/ristsender
  Reason: image not found
~~~

The problem is that all other checks were cancelled, maybe resulting from that failure on arm.

The issue is in the test phase. This formula builds and installs a dylib and a couple of test tools. When running the test tool on Arm, the dylib is not found. It works on Big Sur / Intel and I have no M1 mac to test. How can we debug such issue?

Reply:
@carlocab, is your recent edit `ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath}"` supposed to fix the test issue on Arm?

Reply:
Yes.

Reply:
I had to slightly amend your proposed modification to make it work, namely:
~~~
    ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath}"
    system "meson", "-Dprefix=#{prefix}", "build", ".", "--default-library", "both"
    system "ninja", "install", "-C", "build"
~~~
Hopefully, it will work on Arm.
Could you validate the new commit?
Thanks